src/i18n/zh-cn/ui.ts

```diff
@@ -1,47 +1,64 @@
-import { UIDictionary } from '../translation-checkers';
-
-export default UIDictionary({
```

Comment:
Same here! `UIDictionary` makes sure we don’t accidentally write `site.descripton` instead of `site.description` for example.